src/appleseed/renderer/utility/paramarray.h

```diff
@@ -109,7 +109,7 @@ class APPLESEED_DLLSYMBOL ParamArray
     template <typename T>
     T get_required(
         const char*             name,
-        const T&                default_value) const;
+        const T&                default_value = T()) const;
```

Comment:
Agree with that, but it deserves a separate commit.

Reply:
I'll make a separate commit for that during final rebase